### PR TITLE
Add Gnosis Project

### PR DESCRIPTION
**Pull request by fleupold**
This PR adds a product page for Gnosis. @mewwts  is there some way to preview the page to make sure everything will look good on screen?

### CODE REVIEW

File: pages/gnosis/gnosis.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    link: "https://github.com/gnosis"
+tags:
+  - "Gnosis Safe"
+  - "Gnosis Protocol"
+  - "DEX"
+  - "DeFi"
+  - "IDO"
+dashboards:
+  - "gnosis-safe_2"
+  - "HtS7kMiZWYS2bxA5oArxlevFwX4B0O3ZZAiRxcd3"
```
Comment (mewwts) [inline pages/gnosis/gnosis.md:24]: This doesn't look like it is a dashboard

**Comment by mewwts** [comment]
Thanks you @fleupold! Currently no way to preview, unfortunately, but we're looking at it.

File: pages/gnosis/gnosis.md (hunk 1 above)
Comment (fleupold) [reply pages/gnosis/gnosis.md:24]: It's supposed to be https://explore.duneanalytics.com/public/dashboards/HtS7kMiZWYS2bxA5oArxlevFwX4B0O3ZZAiRxcd3

I think it should also be available under https://explore.duneanalytics.com/dashboard/kpi-gnosis-protocol but when I try to access it via that link I only see "This dashboard is private"

Comment (mewwts) [reply pages/gnosis/gnosis.md:24]: Ah got it! reached out on TG.

**Comment by mewwts** [comment]
merged here https://github.com/duneanalytics/projects/commit/f58824efdb200aa2bcfc2e3aa3e8d20ca44c62a8

**Pull request closed without merging**